avocado/test.py

```diff
@@ -598,6 +598,44 @@ def action(self):
         raise exceptions.TestNotFoundError(e_msg)
 
 
+class BuggyTest(Test):
+
+    """
+    Used when a test could not be imported (possibly an avocado buggy test, but it could be a buggy random py module).
```

Comment:
"Used when a test could not be imported: possibly a buggy test, but it could be just a buggy random module."